tests/testLicense.py

```diff
@@ -4,13 +4,16 @@
 """
 from __future__ import print_function, division
 
-import os, sys, shutil, re, imp, glob
-import unittest, pdb
+import os
+import re
+import unittest
+import pdb  # noqa: F401
```

Comment:
beyond the scope of this PR, but why is this needed?

Reply:
pdb is not used by the code so I have to decide whether to leave it in and mask it, or remove the line on the basis that we don't need to have it there at all. In the interests of minimizing the change I masked it but I'm happy to remove the pdb line completely.

Reply:
(I think the point is that some people like to leave pdb imports hanging around to make life imperceptibly easier when adding a debug breakpoint later on)

Reply:
yeah, but that shouldn't really live in the code similarly to random print statements that sole purpose it to make debugging easier. But this is also a topic for further cleanup.

Reply:
I'll remove it.